cloud/shared/bin/lib/config_loader.py

```diff
@@ -45,20 +47,35 @@ def use_local_backend(self):
     def skip_confirmations(self):
         return os.getenv('SKIP_CONFIRMATIONS', False)
 
-    def load_config(self):
-        self._load_config()
+    def load_config(self, config_file):
+        self._load_config(config_file)
         return self.validate_config()
 
-    def _load_config(self):
+    def _load_config(self, config_file):
+        cwd = os.getcwd()
+        full_config_path = os.path.join(cwd, config_file)
+        if not os.path.exists(full_config_path):
+            exit(f'Cannot find file {full_config_path}')
+        print(f'Getting config from {full_config_path}')
+        command = shlex.split(
+            f'env -i bash -c "source {full_config_path} && env"')
+        proc = subprocess.Popen(command, stdout=subprocess.PIPE)
+        for line in proc.stdout:
+            (key, _, value) = line.decode().partition("=")
+            os.environ[key] = value.strip()
+        proc.communicate()
+
```

Comment:
suggestion (blocking): this could use a comment to describe what's happening, and potentially extracted to a method with a more descriptive method name